### DISCUSSION TITLE
How do I increase video quality?

### DISCUSSION THREAD

**Discussion by timotheeg** [peers/peerjs#1144] [General]
Hi, 

What options or params can I pass to increase video quality? All the peers are on the same LAN and so bandwidth is no issue, and I hope to boost the quality to the max possible.

Thanks!

**Comment by jonasgloning** [comment]
I think this is a MDN article is great start: https://developer.mozilla.org/en-US/docs/Web/Media/Formats/WebRTC_codecs
Supported codecs depend on the browser in use, the version, sometimes to operating system. Some have license restrictions.

Also, see https://developer.mozilla.org/en-US/docs/Web/API/MediaTrackConstraints for options when getting user media (`sampleRate`, `frameRate`, `width/height`).

PeerJS will probably provide some presets in the future, but there’s no recommendation for now.

EDIT: If you found the right settings for max quality, please post it here for future reference! I’ll add them to the documentation.
